src/utils/nanoid.ts

```diff
@@ -0,0 +1,4 @@
+// https://github.com/ai/nanoid/blob/master/nanoid.js
```

Comment:
I use NanoID in other projects, so I am a big fan (quick pseudo-UUID generation, compared with the `uuid` package which we use already in Thorium). However, I would like to suggest importing the NPM package, because otherwise we might miss future updates / bug fixes.

https://www.npmjs.com/package/nanoid

https://www.npmjs.com/package/@types/nanoid